src/panels/lovelace/components/hui-conditional.ts

```diff
@@ -0,0 +1,154 @@
+import { LitElement } from "lit-element";
+import { HomeAssistant } from "../../../types";
+import { LovelaceCard } from "../types";
+import { LovelaceCardConfig, Condition } from "../../../data/lovelace";
+import { LovelaceElement, LovelaceElementConfig } from "../elements/types";
+import { EntityRowConfig } from "../entity-rows/types";
+import { createCardElement } from "../common/create-card-element";
+// import { createRowElement } from "../common/create-row-element";
+
+export interface ConditionalConfig {
+  conditions: Condition[];
+  card: LovelaceCardConfig;
+  elements: LovelaceElementConfig[];
+  rows: EntityRowConfig[];
+  // todo: add row (can't find any createhuirow or LovelaceRowConfig...)
+  // rows: Lovelace;
+  // use it with createRowElement
+}
+
+export class HuiConditional extends LitElement {
```

Comment:
Why is this a `LitElement` ? The original conditional card is derived from an HTMLElement because there is no lit logic needed/used.